docs/source/user-guide/general/drift_guide.rst

```diff
@@ -0,0 +1,292 @@
+* `How Can You Use Deepchecks to Detect Drift? <#how-can-i-use-deepchecks-to-detect-drift>`__
+
+
+What Is Distribution Drift?
+==========================
+
+Distribution drift is a change in the underlying distribution of the data, the label or of the relationship between them. Drift happens over time,
+and can occur either gradually or suddenly.
+
+Common reasons for drift can be natural changes in the data, such as shopping habits changing for an online retailer
+(for example, during the covid pandemic, virtual shopping has risen significantly and buyer patterns have changed), or
+data acquisition and processing issues, such as a camera settings changing accidentally to have lower exposure.
+
+So Why Is Drift So Important?
+-----------------------------
+
+Machine learning models are meant to predict on unseen data, based on previous known data (more accurately, this refers
+to supervised machine learning models, for which drift is most relevant).
+If the data, or the relationships between the different variables and target label, has changed, our model may not be as
+accurate as it was before.
+
+Detecting drift is an important warning sign that our model may be not as accurate on the new data compared to the training data, and that it should be
+adjusted or retrained on different data.
+In production environments, detecting drift (and other measured derived from drift, such as model confidence) is often the only way to know that our model performance is deteriorating,
+as in many cases the label is unknown for some time after the prediction is made.
+
+It is important to note that not all changes in data represent drift. For example, periodic changes in data due to daily, weekly or
+seasonal changes are usually not considered drift, as they as they are often present in the training data.
+
+.. image:: /_static/images/general/patterns-of-concept-drift.png
+   :alt: Different change patterns in data
+   :align: center
+
+Different change patterns in data. `Source <https://www.iosrjournals.org/iosr-jce/papers/Vol17-issue1/Version-2/D017122026.pdf>`_.
+
+Which Types of Drift Are There?
+================================
+
+In machine learning, we usually refer to 3 types of drift:
+
+Concept Drift
+-------------
+Concept drift happens when the underline relation between the data and the label has changed. In layman terms, this means that our
+data has changed in a way that affects our label, meaning a model trained on previous data will have difficulty predicting
+on the new data, as the label can't be predicted in the same way anymore.
+
+Concept drift will almost always require some changes to the model, and will require retraining it on newer data.
+
+Data Drift
+----------
+Data drift is any change in the distribution of the data. It usually refers to changes that do not significantly affect the label.
+However, even when not affecting the model, data drift is important, as it can expose other undetected issues.
```

Comment:
Maybe add in what scenarios does it affect, when does it not.
Maybe say something about areas in the distribution that were under represented in the train and are now more common